posts/migrations/0002_postimage.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.0.2 on 2022-03-02 23:09
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('posts', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='PostImage',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('blob', models.ImageField(max_length=255, upload_to='BlobStorage')),
+            ],
```

Comment:
Is this model autogenerated?

Reply:
model that I added because I thought I needed it and then deleted it... there's another migration that deletes it... admittedly a bit unfamiliar with how the migration process works

Reply:
Ah gotcha, we'll leave it for now then. I'll squash the migrations just before we merge